test/uk/gov/hmrc/tradergoodsprofilesrouter/service/RequestAdviceServiceSpec.scala

```diff
@@ -66,7 +69,7 @@ class RequestAdviceServiceSpec
 
     when(getRecordService.fetchRecord(any, any, any)(any))
       .thenReturn(Future.successful(Right(getResponseDataWithAdviceStatus())))
-
+    when(appConfig.hawkConfig.getRecordsUrl).thenReturn("/url")
```

Comment:
I think it would be good here to verify that we call the appConfig.pagaCnf.getRecordUrl, and also verify that the connector get called with that url